deployment_helper/templates/run_script_on_host.sh

```diff
@@ -29,7 +29,14 @@ function is_local()
 	return
     fi
     host=$(curl -s $AWS_PUBLIC_HOSTNAME_URL)
-    if [ $cmd_host == $host ]; then
+    # Catch curl failure
+    rc=$?
+    if [ $rc != 0 ]; then
+	echo 1
```

Comment:
I do not think that we want to conclude that the host is local (echo 1) if the curl command fails.
I am committing a fix for these issues in my latest CR that looks as below:
```
    host=$(curl -s -m 1 $AWS_PUBLIC_HOSTNAME_URL)
    if [ "$cmd_host" == "$host" ]; then
        echo 1
        return
    fi
```

This does not look at the curl return code explicitly, but it makes sure that the curl command returns in 1 second and that the empty value for $host is handled correctly. I have tested this on a non-aws host and it works as expeted.

Reply:
Fair point. I guess I was thinking more of the error behaviour.